tests/func_test.py

```diff
@@ -14,23 +14,27 @@ class BasicInstallTest(unittest.TestCase):
     def setUp(self):
         options = Options()
         options.add_argument("--headless")
-        self.browser = webdriver.Chrome(options=options)
+        options.add_argument('--no-sandbox')
+        options.add_argument('--window-size=1920,1080')
+        options.add_argument('--disable-gpu')
+        self.browser = webdriver.Remote(command_executor='http://selenium:4444/wd/hub', options=options)
+        time.sleep(5)
```

Comment:
looks like crutch

Why shall we sleep for 5 seconds?
Can we somehow replace it with some async call?

Reply:
please ignore